README.md

```diff
@@ -13,11 +13,11 @@ Talawa is a modular open source project to manage group activities of both non-p
 
 Core features include:
 
-1.  Membership management
-2.  Groups management
-3.  Event registrations
-4.  Recurring meetings
-5.  Facilities registrations
+1. Membership management
+2. Groups management
```

Comment:
1. Please revert all changes to this file. 
2. It is out of scope of the issue. 
3. The changes impede our ability to have the code quality we require as a project